modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/service/impl/BatchEngineTaskLocalServiceImpl.java

```diff
@@ -52,14 +53,24 @@ public BatchEngineTask addBatchEngineTask(
 		batchEngineTask.setBatchSize(batchSize);
 		batchEngineTask.setCallbackURL(callbackURL);
 		batchEngineTask.setClassName(className);
-		batchEngineTask.setContent(
-			new OutputBlob(
-				new UnsyncByteArrayInputStream(content), content.length));
+
+		if (content == null) {
```

Comment:
@ivicac when will content be null? For export?
When you started this, there was no exported mentioned, but now it seems like we are putting both import/export in the same entity and service, which is wrong.

We should consider to separate it to BatchEngineImportTask and BatchEngineExportTask.